src/checks_linux.go

```diff
@@ -21,6 +21,10 @@ func processCheck(check *check, checkType string, arg1 string, arg2 string, arg3
 		}
 		result, err := guestDisabledLDM()
 		return err == nil && !result
+	case "PasswordChanged":
+		if check.Message == "" {
+			check.Message = "Insecure password has been changed"
```

Comment:
Add username here

Reply:
Can't that already be in the check message?

Reply:
This is default message, so if person forgets and puts multiple it can get confusing

Reply:
Done